monitoring/statistics_impl.h

```diff
@@ -131,6 +131,12 @@ inline void RecordTick(Statistics* statistics, uint32_t ticker_type,
                        uint64_t count = 1) {
   if (statistics) {
     statistics->recordTick(ticker_type, count);
+    if (ticker_type == ERROR_HANDLER_BG_ERROR_COUNT ||
```

Comment:
When we clean up the "_MISSPELLED" tickers, it's not very obvious this code needs to be taken care of too. Without that, it will cause counting to unintended tickers. I know it's more duplicated code, but what about adding the ticker incrementing code to the source of when these tickers's counter parts get incremented in `error_handler.cc`. That way, the clean up is purely removing a couple of lines.

BTW, do you mind adding an entry for this change to `unreleased_history/public_api_changes` directory, you basically need to create a separate `.md` file, e.g. fix_ticker_typo.md into the directory, and describe the change in the file.